src/models/model.cpp

```diff
@@ -723,8 +723,15 @@ void model::setup(size_t max_mini_batch_size,
 
   setup_layer_topology();
   setup_layer_execution_order();
+  setup_layer_grid_tags(grids_);
+  
   if (this->is_subgraph_parallelism_enabled()) {
-    setup_subgrids();
+  	for (auto& layer: this->get_layers())
```

Comment:
please run clang-format on your code. See [CONTRIBUTING.md](https://github.com/LLNL/lbann/blob/develop/CONTRIBUTING.md) for more information.